tests/helpers.js

```diff
@@ -85,7 +85,11 @@ class FuzzTest {
 		return this;
 	}
 
-	executeWithCli() {
+	executeAsync() {
```

Comment:
Can we rename this to something like `spawn`, as `async` already has multiple meanings in our context.